dags/pytorch_xla/configs/pytorchxla_torchbench_config.py

```diff
@@ -115,7 +115,10 @@ def get_torchbench_tpu_config(
       task_owner=test_owner.PEI_Z,
   )
 
-  job_metric_config = metric_config.MetricConfig(use_runtime_generated_filename=True)
+  job_metric_config = metric_config.MetricConfig(
+      json_lines=metric_config.JSONLinesConfig("benchmark/metric_report.jsonl"),
```

Comment:
Nit: you don't need to include `benchmark` right? we could just use benchmark_id to distinguish each test.

Reply:
Yes, in this config, it doesn't matter.